server/routes/banner.js

```diff
@@ -9,8 +9,13 @@ const PUBLIC_PATH = `${__dirname}/../../dist/`;
 /**
  *  Returns all banners
  */
-router.get("/", ensureAuthenticated, (req, res) => {
-  res.json(bannerCtrl.getBannersArray());
+router.get("/", ensureAuthenticated, async (req, res) => {
+  try {
+    const banners = await bannerCtrl.getBannersArray();
+    res.json(banners);
+  } catch(e) {
+    throw new Error(e);
```

Comment:
Préférer renvoyer une `500` sans trop d'information par problématique de sécurité

Reply:
Done in 598bac694a5bc952b4ce5c34b0382652cc9884e1